### ISSUE TITLE
git-repo-age rule fails on Travis

### ISSUE THREAD

**Issue by carlosperate** [sindresorhus/awesome-lint#45]
Can be seen here: https://travis-ci.org/carlosperate/awesome-microbit/builds/427309076

A repository hosted on GitHub (https://github.com/carlosperate/awesome-microbit/) and the travis script doesn't have anything complex going on: 
https://github.com/carlosperate/awesome-microbit/blob/f2d3fae2faf85f4e4022ffa8057870a7c48912f9/.travis.yml
```yml
language: node_js
node_js:
  - 'node'

before_install:
  - rvm install 2.2.2

install:
  - npm install --global awesome-lint
  - gem install awesome_bot

script:
  - awesome-lint README.md
  - awesome_bot README.md --allow-redirect --allow-dupe
```

Travis log outputs:

```
$ awesome-lint README.md
- Linting
✖ Linting
README.md:333:3
✖1:1  Awesome list must reside in a valid git repositoryawesome/git-repo-age
```

**Comment by IssuehuntBot** [comment]
@issuehunt has funded $20.00 to this issue.

---
- Submit pull request via [IssueHunt](https://issuehunt.io/repos/64659785/issues/45) to receive this reward.
- Want to contribute? Chip in to this issue via [IssueHunt](https://issuehunt.io/repos/64659785/issues/45).
- Checkout the [IssueHunt Issue Explorer](https://issuehunt.io/issues) to see more funded issues.
- Need help from developers? [Add your repository](https://issuehunt.io/users/44827199/repositories) on IssueHunt to raise funds.

**Comment by chinesedfan** [comment]
It is not a problem of awesome-line, and I created a PR(https://github.com/chinesedfan/awesome-microbit/pull/1) to explain it and pasted to IssueHunt.

**Comment by sindresorhus** [comment]
@chinesedfan Can you do a PR to document that in the readme? Maybe under a “CI setup” section or something.

**Comment by chinesedfan** [comment]
@sindresorhus Done with #67.

**Comment by IssuehuntBot** [comment]
@sindresorhus has rewarded $18.00 to @chinesedfan. [See it on IssueHunt](https://issuehunt.io/repos/64659785/issues/45)

- :moneybag: Total deposit: $20.00
- :tada: Repository reward(0%): $0.00
- :wrench: Service fee(10%): $2.00
